Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -519,11 +519,12 @@ class BrowserViewController: UIViewController {
         super.viewDidAppear(animated)
         log.debug("BVC done.")
 
-        if profile.prefs.stringForKey(LatestAppVersionProfileKey) != AppInfo.appVersion && DeviceInfo.hasConnectivity() {
-            if let whatsNewURL = SupportUtils.URLForTopic("new-ios") {
-                self.openURLInNewTab(whatsNewURL)
-                profile.prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
-            }
+        if let whatsNewURL = SupportUtils.URLForTopic("new-ios"),
+           let latestMajorAppVersion = profile.prefs.stringForKey(LatestAppVersionProfileKey)?.componentsSeparatedByString(".").first
```

Comment:
I think this won't open the what's new page for users upgrading from 1.4 to 2.1 because `LatestAppVersionProfileKey` will be nil. Probably want to do a nil check as well to show the page. There are three cases we'll find: a user just installed the app in which we do not show What's New, a user upgraded it from 2.0 so they already have the pref set, and a user who upgraded from a version less than 2.0 so they do not have the pref set.